pyodide_build/pypabuild.py

```diff
@@ -5,6 +5,16 @@
 import sys
 import traceback
 from collections.abc import Callable, Iterator, Mapping, Sequence
+
+
+# A helper function from pypa/build/__main__.py since
+# it's not in the vendorized code we have
+# TODO: we should move this to a new file. it's out of place
+# between the other imports ;-)
+def _format_dep_chain(dep_chain: Sequence[str]) -> str:
+    return " -> ".join(dep.partition(";")[0].strip() for dep in dep_chain)
```

Comment:
Maybe you can put it in `common.py`.

We need to reorganize common.py too, as it contains too much code anyway.

Reply:
I agree. For the time being, I addressed this in 0aed06b6b02ba02dd08d9686a7c00d4e54437129.